addon/routes/gis-search-form.js

```diff
@@ -13,6 +13,11 @@ import { Query } from 'ember-flexberry-data';
   @extends <a href="http://emberjs.com/api/classes/Ember.Route.html">Ember.Route</a>
 */
 export default Ember.Route.extend({
+
+  model() {
+    return this._getQuery('new-platform-flexberry-g-i-s-map', 'MapL', null, null, null);
```

Comment:
magic strings.
I dont understand for which purposes you declare tabSettings instead of two separate group for map and metadata